DigitalLearningSolutions.Data/Models/User/ChooseACentreAccountViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace DigitalLearningSolutions.Data.Models.User
+{
+    using DigitalLearningSolutions.Data.Enums;
+
+    public class ChooseACentreAccountViewModel
+    {
+        public ChooseACentreAccountViewModel(
+            int centreId,
+            string centreName,
+            bool isCentreActive,
+            bool isActiveAdmin,
+            bool isDelegate,
+            bool isDelegateApproved,
+            bool isDelegateActive
+        )
+        {
+            CentreId = centreId;
+            CentreName = centreName;
+            IsCentreInactive = !isCentreActive;
+            IsActiveAdmin = isActiveAdmin;
+            IsDelegate = isDelegate;
+            IsApprovedDelegate = IsDelegate && isDelegateApproved;
+            IsInactiveDelegate = IsDelegate && !isDelegateActive;
+        }
+
+        public int CentreId { get; }

```

Comment:
Missed this - the conventional way to do a readonly property is to remove the "{ get; }" and add `readonly` before the type declaration. It does make it a bit quicker to take in.

Reply:
Interesting...we have a number of other places in the code where the style I have used is present (I make it 485).

Reply:
It's not a big deal like.